modules/efk-logging-uninstall-efk-ops.adoc

```diff
@@ -3,14 +3,14 @@
 // * logging/efk-logging-uninstall.adoc
 
 [id='efk-logging-uninstall-efk-ops_{context}']
-= Uninstall the Ops Cluster in {product-title}
+= Uninstall the infra cluster in {product-title}
```

Comment:
What's the infra cluster? Is that actually a Real Component Name? If not, I'd s/infra/infrastructure.

Reply:
This section is coming out in another PR. 
The infra cluster was called the Ops Cluster:
https://docs.openshift.com/container-platform/3.5/install_config/aggregate_logging.html#aggregated-ops